### ISSUE TITLE
Rofi theme style not appearing as shown in screenshots

### ISSUE THREAD

**Issue by the-merky** [junnunkarim/dotfiles-linux#7]
Hi! Rofi appears to be just the vanilla rofi(maybe I am wrong it just doesn't look like in the screenshots). The colors work properly when switching themes, but it has rounded corners and shadow on the buttons. I couldn't capture a screenshot right now, but I will add one ASAP. I reinstalled the rofi AUR package and copied the config files over from the newest commit but still no effect. Thanks in advance!

**Comment by junnunkarim** [comment]
@ShambuKumar Sorry about that.
The dwm rice you came across was actually my very first attempt at creating a rice. After that, I shifted to Hyprland, then Qtile, and finally settled on awesomewm. As a result currently this repo right now is somewhat unorganized.

I do have plans to make this repo a central hub for all my dotfiles, so that people can easily find and install the prefered configs. But currently, I am busy with my bachelor and stuffs. So it might take some time.

Anyway, I have restored the rofi configuration to its previous appearance. It should now match the screenshot as before.
